spec/inputs/syslog_spec.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
```

Comment:
This is a custom spec_helper only for this plugin, same thing is in some other plugins, for example the pattern core.